File: Practica3/Marco teorico/1. Modelamiento de Clases en POO/MOD1.java
class Persona {
    private int edad;
    private String nombre;

    public Persona(int edad, String nombre) {
        this.edad = edad;
        this.nombre = nombre;
    }

    public String getNombre() {
        return this.nombre;
    }

    public int getEdad() {
        return this.edad;
    }
}

public class Main {
    public static void main(String[] args) {
        Persona p1 = new Persona(25, "Juan");
        System.out.printf("La edad de %s es %d%n", p1.getNombre(), p1.getEdad());
    }
}

File: Practica3/Marco teorico/1. Modelamiento de Clases en POO/MOD1.py
class Persona:
    def __init__(self, edad, nombre):
        self.edad = edad
        self.nombre = nombre
    
    def getEdad(self):
        return self.edad
    
    def getNombre(self):
        return self.nombre

p1 = Persona(25, "Juan")
print(f"La edad de {p1.getNombre()} es {p1.getEdad()}")